utils/evaluate/evaluate.go

```diff
@@ -1,13 +1,29 @@
 package evaluate
 
 import (
+	"strconv"
+
 	"github.com/antonmedv/expr"
 )
 
 // EvalCondition evaluates the condition with the resultValue as an input
 func EvalCondition(resultValue interface{}, condition string) (bool, error) {
 	env := map[string]interface{}{
 		"result": resultValue,
+		"asInt": func(in string) int64 {
+			inAsInt, err := strconv.ParseInt(in, 10, 64)
+			if err == nil {
+				return inAsInt
+			}
+			panic(err)
+		},
+		"asFloat": func(in string) float64 {
+			inAsFloat, err := strconv.ParseFloat(in, 64)
+			if err == nil {
+				return inAsFloat
+			}
+			panic(err)
+		},
```

Comment:
This is definitely in line with what I was thinking! For the error handling, we should add a recover at line 28 to catch these panics. From there, we can return an error saying `Unable to convert to Float` and capture that in the status. Otherwise, the panic will break the rest of the analysis run. I think it will make the comparisons more explicit for the users and gives them flexibility if needed.

Reply:
This should also be a good candidate for some unit tests using the assert.Panic function

Reply:
Sounds good! I'll add that in later today 👍

Reply:
So, I added tests, and I did also add the recover clause. But it may be unecessary after playing around with the evaluation library. From the looks of it, it already wraps panics from internal functions as errors with nice pretty print debug messages. 
```
got expected error: strconv.ParseFloat: parsing "NotANum": invalid syntax (1:9)
        	            	 | asFloat(result) == 1.1
        	            	 | ........^
```